src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -91,6 +84,7 @@ export default function Autocomplete({
   return (
     <MUIAutocomplete
       disabled={disabled}
+      readOnly={readOnly}
```

Comment:
Does this work? I tested on story book and it doesn't make any change

Reply:
yes i noticed it doesn't seem to do anything.. it's in the API though https://mui.com/material-ui/api/autocomplete/
i guess i can remove it